ui/src/main/java/edu/wpi/grip/ui/OperationListController.java

```diff
@@ -0,0 +1,103 @@
+package edu.wpi.grip.ui;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.eventbus.Subscribe;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.ui.annotations.ParametrizedController;
+import edu.wpi.grip.ui.util.ControllerMap;
+import edu.wpi.grip.ui.util.SearchUtility;
+import javafx.beans.InvalidationListener;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.MapChangeListener;
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.control.Tab;
+import javafx.scene.layout.VBox;
+
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * Controller for a VBox of {@link OperationController}s.  The user data for this should be what category it shows,
+ * and the filterText property allows for fuzzy searching the operations.
+ *
+ * @see edu.wpi.grip.core.Operation.Category
+ */
+@ParametrizedController(url = "OperationList.fxml")
+public class OperationListController {
+
+    @FXML private Tab root;
+    @FXML private VBox operations;
+
+    @Inject private OperationController.Factory operationControllerFactory;
+    @Inject private Logger logger;
+
+    private final StringProperty filterText = new SimpleStringProperty(this, "filterText", "");
+    private String baseText = null;
```

Comment:
The scope of the field is greater than it needs to be. You can make the scope of the field the method exclusively.